offchain-modules/packages/x/src/monitor/relayer-metric.ts

```diff
@@ -0,0 +1,93 @@
+import * as Prometheus from 'prom-client';
+import { Config, forceBridgeRole } from '../config';
+import { logger } from '../utils/logger';
+
+export type chainType = 'ckb' | 'eth';
+export type txType = 'ckb_mint' | 'ckb_burn' | 'eth_lock' | 'eth_unlock';
+export type txStatus = 'success' | 'failed';
+export type txTokenInfo = {
+  token: string;
+  amount: number;
+};
+
+export class RelayerMetric {
+  private static _relayBlockHeightNum: Prometheus.Gauge<any>;
+  private static _relayBridgeTxNum: Prometheus.Counter<any>;
+  private static _relayBridgeTokenAmountNum: Prometheus.Gauge<any>;
+
+  private static _gateway: Prometheus.Pushgateway;
+  private static _openPushMetric: boolean;
+
+  init(role: forceBridgeRole, pushGatewayURL: string): void {
+    const register = new Prometheus.Registry();
+
+    RelayerMetric._relayBlockHeightNum = new Prometheus.Gauge({
```

Comment:
`static init(pushGatewayURL: string)`
`constructor(role: Role)`

we should avoid change the static items in an instance method, it's easy to be confused